services/subscriber/service.go

```diff
@@ -183,20 +183,28 @@ func (s *Service) createSubscription(se subEntry, mode string, destinations []st
 	default:
 		return nil, fmt.Errorf("unknown balance mode %q", mode)
 	}
-	writers := make([]PointsWriter, len(destinations))
-	stats := make([]writerStats, len(writers))
-	for i, dest := range destinations {
+	writers := []PointsWriter{}
```

Comment:
I think using a slice literal is cleaner and easier to read. It is my default unless there is a clear performance reason to do otherwise.

Checking now I notice that using make with a zero length is a common pattern in the InfluxDB codebase. I'll change these lines.